ecommerce-ui/src/GetPropertyData.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import AddToCart from './AddToCart.js';
+
+class GetPropertyData extends React.Component {
+
+  render () {
+    return (
+      this.props.propertyData.map((property, i) => {
+        return (
+          <div className="wrapper" key={i}>
+            <div className="image">
+              <img src={property.image} alt="" height='250' width='400'/>
+            </div>
+            <div className="location">{property.location.city},{' '}
+              {property.location.country}
+            </div>
+            <div className="description">{property.houseType}</div>
+            <div className="rating">{property.rating.stars}\5{' '}Stars
```

Comment:
I would just use string interpolation instead of dropping in these spaces.
```suggestion
            <div className="rating">`${property.rating.stars}\5 Stars`
```